### ISSUE TITLE
Only get Jira issues that are new

### ISSUE THREAD

**Issue by joncodo** [merico-dev/lake#305]
From the old code

```
const latestUpdated = await issuesCollection.find().sort({ 'fields.updated': -1 }).limit(1).next()
  const $addToSet = { boardIds: boardId }
  let jql = ''
  if (!forceAll && latestUpdated) {
    const jiraDate = dayjs(latestUpdated.fields.updated).format('YYYY/MM/DD HH:mm')
    jql = encodeURIComponent(`updated >= '${jiraDate}' ORDER BY updated ASC`)
  }
  let counter = 0

```

**Comment by klesh** [comment]
Yup, I' working on it at end of the week

**Comment by klesh** [comment]
Closed by #349
